src/app/ReadHandler.cpp

```diff
@@ -93,41 +88,56 @@ ReadHandler::ReadHandler(ManagementCallback & apCallback, Observer * observer) :
     mObserver = observer;
 }
 
-void ReadHandler::ResumeSubscription(CASESessionManager & caseSessionManager,
-                                     SubscriptionResumptionStorage::SubscriptionInfo & subscriptionInfo)
+void ReadHandler::OnSubscriptionResumed(const SessionHandle & sessionHandle, SubscriptionResumptionSessionEstablisher & helper)
```

Comment:
rename `helper` to something else. looks like `resumer` or `resumption_session_establisher` or similar.

Reply:
Done